internal/confmapprovider/discovery/bundle/bundle.d/receivers/postgresql.discovery.yaml.tmpl

```
@@ -0,0 +1,73 @@
+{{ receiver "postgresql" }}:
+  enabled: true
+  rule:
+    docker_observer: type == "container" and any([name, image, command], {# matches "(?i)postgres"}) and not (command matches "splunk.discovery")
+    host_observer: type == "hostport" and command matches "(?i)postgres" and not (command matches "splunk.discovery")
+    k8s_observer: type == "port" and pod.name matches "(?i)postgres"
+  config:
+    default: {}
```

I think required fields that aren't populate by endpoint details should use the discovery default:

```suggestion
    default:
        username: {{ defaultValue }}
        password: {{ defaultValue }}
```